specs/protocol/isthmus/derivation.md

```diff
@@ -51,3 +52,11 @@ cast keccak $(cast concat-hex 0x000000000000000000000000000000000000000000000000
 
 This transaction MUST deploy a contract with the following code hash
 `0x6e49e66782037c0555897870e29fa5e552daf4719552131a0abce779daec0a5d`.
+
+# Span Batch Updates
+
+[Span batches](../delta/span-batches.md) are a span of consecutive L2 blocks than are batched submitted.
+
+Span batches contain the L1 transactions and transaction types that are posted containing the span of L2 blocks.
+Since [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702) introduces a new transaction type, the Span Batch must
+be updated to support the [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702) transaction.
```

Comment:
Is there a precise specification of how SetCode transactions are encoded in a span batch somewhere already?

Reply:
Ideally this spec contains the exact serialization of a span batch before its merged

Reply:
Updated. Let me know if this is what you're looking for :)